qa-tests-backend/src/main/groovy/util/SplunkUtil.groovy

```diff
@@ -62,26 +59,16 @@ class SplunkUtil {
         return syslogStrings
     }
 
-    static List<SplunkAlert> waitForSplunkAlerts(int port, int timeoutSeconds) {
-        int intervalSeconds = 3
-        int iterations = timeoutSeconds / intervalSeconds
+    static List<SplunkAlert> waitForSplunkAlerts(int port, String search = "search") {
+        log.info("Waiting for data to arrive in Splunk for search query: " + search)
         List results = []
-        Exception exception = null
-        Timer t = new Timer(iterations, intervalSeconds)
-        while (results.size() == 0 && t.IsValid()) {
-            def searchId = null
-            try {
-                searchId = createSearch(port)
-                exception = null
-            } catch (Exception e) {
-                exception = e
-            }
+        withRetry(40, 15) {
+            def searchId
+            searchId = createSearch(port, search)
```

Comment:
Question: do we need to create a new search on every retry?
Would it be possible to create the search once and then run `getSplunkAlerts` in the retry-loop?

Reply:
From what I can find in the API docs, a new search needs to be created for every try